core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1009,7 +1017,7 @@ static function ($allowedTag)
 
 						if ($objFile !== null)
 						{
-							$arrCache[$strTag] = System::urlEncode($objFile->path);
+							$arrCache[$strTag] = System::urlEncode(System::getContainer()->get('contao.assets.files_context')->getStaticUrl() . $objFile->path);
```

Comment:
```suggestion
							$arrCache[$strTag] = System::getContainer()->get('contao.assets.files_context')->getStaticUrl() . System::urlEncode($objFile->path);
```
Otherwise this would double encode the static url part.

Reply:
Good catch! 🦊